packages/grid_client/src/clients/tf-grid/contracts.ts

```diff
@@ -208,7 +216,35 @@ class TFContracts extends Contracts {
       throw err;
     }
   }
+  /**
+   * Get contract discount package
+   * @param {GetDiscountPackageOptions} options
+   * @returns {Promie<DiscountLevel>}
+   */
+  async getDiscountPackage(options: GetDiscountPackageOptions): Promise<DiscountLevel> {
+    const gqlClient = new Graphql(options.graphqlURL);
+    const body = `query getConsumption($contractId: BigInt!){
+            contractBillReports(where: {contractID_eq: $contractId}, limit: 1 ) {
```

Comment:
i think this will make the first discount package will always displayed, i suggest to remove limit and get the last element in the list this may fix the issue